libs/common/src/helpers/with-boolean-feature-flag.tests.ts

```diff
@@ -0,0 +1,42 @@
+const originalEnv = process.env;
+
+export const withBooleanFeatureFlagTests = ({ featureFlagName, getFeatureFlagValue }: { featureFlagName: string; getFeatureFlagValue: () => boolean }) => {
+  describe(`${featureFlagName}`, () => {
+    afterAll(() => {
+      process.env = { ...originalEnv };
```

Comment:
Just had a (slightly worrying, given I've done the same thing) thought - will this actually reset the process.env?

I think we need to spread
` const originalEnv = { ...process.env };`

Reply:
Fixed here, thanks for fixing elsewhere!